README.md

```diff
@@ -14,7 +14,16 @@ The docs output is generated by a CI pipeline which means that we can update the
 
 During the documentation build process, each PR goes through Vercel CI to test the doc updates and build a preview of your changes. Vercel for GitHub automatically deploys and provides preview deployment URLs. After the PR status is ready, click **Visit Preview** and navigate to the changed page to see your doc updates.
 
-Sometimes it is helpful to see local updates before you push changes to GitHub. To run a local build:
+Sometimes it is helpful to see local updates before you push changes to GitHub.
+<!-- In case of version change, update .github/workflows/{test-tutorials.yml, snarkjs-api-reference.md} accordingly -->
+Local builds require NPM, version 16 precisely.
```

Comment:
I think you mean node instead of NPM here right?

Reply:
Updated, thanks.